tools/PI/DevHome.PI/Pages/WERPage.xaml.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using DevHome.Common.Extensions;
+using DevHome.PI.Models;
+using DevHome.PI.Telemetry;
+using DevHome.PI.ViewModels;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Navigation;
+
+namespace DevHome.PI.Pages;
+
+public sealed partial class WERPage : Page
+{
+    private WERPageViewModel ViewModel { get; }
+
+    public WERPage()
+    {
+        ViewModel = Application.Current.GetService<WERPageViewModel>();
+        InitializeComponent();
+    }
+
+    protected override void OnNavigatedTo(NavigationEventArgs e)
+    {
+        base.OnNavigatedTo(e);
+        Application.Current.GetService<TelemetryReporter>().SwitchTo(Feature.WERReports);
+    }
+
+    private void SelectorBar_SelectionChanged(SelectorBar sender, SelectorBarSelectionChangedEventArgs args)
+    {
+        UpdateInfoBox();
+    }
+
+    private void WERDataGrid_SelectionChanged(object sender, SelectionChangedEventArgs e)
+    {
+        UpdateInfoBox();
+    }
+
+    private void UpdateInfoBox()
+    {
+        if (WERDataGrid.SelectedItem is null)
+        {
+            WERInfo.Text = string.Empty;
+            return;
+        }
+
+        SelectorBarItem selectedItem = InfoSelector.SelectedItem;
```

Comment:
Aren't we encouraged to use var in these circumstances?

Reply:
I suspect there is falls into the same bucket as tabs vs spaces. Since it didn't give me a complier error, I'm rolling with this. :)